praw/__init__.py

```diff
@@ -1125,6 +1126,17 @@ def get_top(self, *args, **kwargs):
         """
         return self.get_content(self.config['top'], *args, **kwargs)
 
+    def get_traffic(self, subreddit):
+        """Return the traffic stats for a subreddit.
```

Comment:
Given that this endpoint returns json, we may want to specify that it returns json rather than a "TrafficStats" object.